nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -796,6 +796,12 @@ def __init__(
         drop_path_rate: float = 0,
         use_emha: bool = False,
         ub_tp_comm_overlap: bool = False,
+        ub_bulk_wgrad: bool = True,
```

Comment:
The changes here are already in NeMo main (see [here](https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/nlp/modules/common/megatron/transformer.py#L799)), looks like you'd need to rebase your branch.
Otherwise the PR looks good to me.